doc/installation.rst

```diff
@@ -23,9 +19,9 @@ To upgrade, execute
 
 .. code:: bash
 
-    $ conda update garnett
+    $ conda install -c conda-forge garnett
```

Comment:
I think you replaced the section on _updating_ garnett instead of the section on _installing_ garnett. These edits should go in the installation section.

Reply:
Whoops! fixed